app/controllers/application_pets_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ApplicationPetsController < ApplicationController
+    def index
```

Comment:
You're on the right track creating this joins controller! But you don't necessarily need an index here (and it doesn't look like this is being used)

A great use of this controller is to house the `create` and `update` actions for PetApplications (i.e. joins records). 

When a user adds a pet to their application, it could direct to a create action in this controller in which you create the joins record. Then it would redirect back to the user's applications show page

When an admin clicks "approve" or "deny", it could direct to an update action here that could update the status of that joins record! THen, it would redirect back to the admin application show page - `redirect_to "admin/applications/#{application.id}"`